FreeRTOS-Plus/Source/Application-Protocols/network_transport/mbedtls_pk_pkcs11.c

```diff
@@ -111,7 +175,6 @@ static int p11_ecdsa_check_pair( const void * pvPub,
 static void p11_ecdsa_debug( const void * pvCtx,
                              mbedtls_pk_debug_item * pxItems );
 
-/* Helper functions */
 static int prvEcdsaSigToASN1InPlace( unsigned char * pucSig,
```

Comment:
why are these functions / several below not documented with doxygen?

Reply:
The names a pretty self explanatory :)